the patch below adds <cstdlib> in defs.h to provide the missing prototypes to strto* functions (build fails under gcc 4.4 HEAD), and also removes the <string.h> include, which is already included in <string>.

Index: src/plugins/contrib/help_plugin/defs.h
===================================================================
--- src/plugins/contrib/help_plugin/defs.h	(revision 5177)
+++ src/plugins/contrib/help_plugin/defs.h	(working copy)
@@ -13,7 +13,7 @@
 #include <ctime>
 #include <cctype>
 #include <cstdio>
-#include <string.h>
+#include <cstdlib>
 
 #define kDebug(x) DummyOutput()
 #define kWarning(x) DummyOutput()


